Sources/Sentry/SentryEvent.m

```diff
@@ -19,6 +19,7 @@
 #import <Sentry/NSDate+SentryExtras.h>
 #import <Sentry/NSDictionary+SentrySanitize.h>
 #import <Sentry/SentryMeta.h>
+#import "SentryEnvelope.h"
```

Comment:
This is wrong here, should go in the `#else` and here we need `#import <Sentry/SentryEnvelope.h>`

Reply:
Good catch, that shouldn't be here 👍